lib/src/main/java/com/thenewboston/api/bankapi/datasource/BankDataSource.kt

```diff
@@ -125,4 +130,39 @@ class BankDataSource @Inject constructor(private val networkClient: NetworkClien
             else -> Outcome.Success(blocks)
         }
     }
+
+    suspend fun updateAccount(
+        accountNumber: String,
+        trustLevel: Double,
+        nodeIdentifier: String,
+        signature: String
+    ): Outcome<Account> = makeApiCall(
+        call = { doUpdateAccount(accountNumber, trustLevel, nodeIdentifier, signature) },
+        errorMessage = "Could not update trust level of given account"
+    )
+
+    private suspend fun doUpdateAccount(
+        accountNumber: String,
+        trustLevel: Double,
+        nodeIdentifier: String,
+        signature: String
```

Comment:
Why don't we allow the SDK user to pass in the request body instead of introducing 4 params?

Reply:
That would indeed clean up the API here but also make the SDK a bit less convenient to use IMHO. If that's fine, I'll refactor it to accepting a request body :+1:

Reply:
Once you document the usage of the SDK it should be fine since (which will we will do once we finish the bank API). They should using the data class request object we created for them.